File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/VoucherSummary.java
package com.cinema.ticket_booking.data.model.response;

import com.google.gson.annotations.SerializedName;

public class VoucherSummary {
    @SerializedName("code")
    private String code;
    @SerializedName("description")
    private String description;
    @SerializedName("discountType")
    private String discountType;
    @SerializedName("discountValue")
    private double discountValue;
    @SerializedName("maxDiscount")
    private double maxDiscount;
    @SerializedName("minOrder")
    private double minOrder;
    @SerializedName("endDate")
    private String endDate;
    @SerializedName("status")
    private String status;

    public String getCode() {
        return code;
    }

    public String getDescription() {
        return description;
    }

    public String getDiscountType() {
        return discountType;
    }

    public double getDiscountValue() {
        return discountValue;
    }

    public double getMinOrder() {
        return minOrder;
    }

    public double getMaxDiscount() {
        return maxDiscount;
    }

    public String getEndDate() {
        return endDate;
    }

    public String getStatus() {
        return status;
    }
}

File: App/app/src/main/java/com/cinema/ticket_booking/data/model/response/VoucherSummary.kt
package com.cinema.ticket_booking.data.model.response

import com.google.gson.annotations.SerializedName

data class VoucherSummary(
    @SerializedName("code")
    val code: String?,

    @SerializedName("description")
    val description: String?,

    @SerializedName("discountType")
    val discountType: String?,

    @SerializedName("discountValue")
    val discountValue: Double,

    @SerializedName("maxDiscount")
    val maxDiscount: Double,

    @SerializedName("minOrder")
    val minOrder: Double,

    @SerializedName("endDate")
    val endDate: String?,

    @SerializedName("status")
    val status: String?
)
